### ISSUE TITLE
version 1.3.0 hangs

### ISSUE THREAD

**Issue by rtelson** [microsoft/AzureStorageExplorer#403]
**Storage Explorer Version**:   1.3.0
**Platform/OS Version**:  Window 10
**Architecture**: 
**Regression From**: 

#### Steps to Reproduce: ####
1.   Azure Storage Explorer
2.  Open Storage
3. Move files

#### Expected Experience: ####
 Azure Storage Explorer goes through the list of files for about 3-5 minutes, then hands

#### Actual Experience: ####
 Cannot exit, change to another step or view. Must kill the task.

**Comment by rtelson** [comment]
Version 1.3.0 (newest release) of Azure Storage Explorer very unstable.

**Comment by JasonYeMSFT** [comment]
Do you mean the splash screen hangs when Storage Explorer starts? Does it happen in the old versions? If possible, please fill the template so we can have more information to investigate. Thanks.

**Comment by rtelson** [comment]
The splash screen is fine. This happens when we start moving files onto the platform. We did not have this problem with 1.2.0, and it started happening immediately after upgrading to 1.3.0. It has been consistent to the point the process is not moving forward. We would like to roll back; we need you help understand how, if that is possible. It would appear that we cannot use 1.3.0.

**Comment by jinglouMSFT** [comment]
Sorry to hear about that. I would like to work with you on diagnosing the issue. However, to unblock you first, you can download 1.2.0 from the Azure Storage Explorer download page here:  https://azure.microsoft.com/en-us/features/storage-explorer/
Go to the bottom of that page. There is a section called "Previous version".  The bits for 1.2.0 are there.  You can just down it and install it.  This will replace your current version installed on the machine.

**Comment by rtelson** [comment]
Jason: I believe the error was on our side. We downgraded to 1.2.0 and got the same results. Further analysis led us to a video file (.mov) as the cause. We then re-installed 1.3.0 and it is hanging while uploading the same file as the 1.2.0 version. 

We’re looking further into it.

**Comment by MRayermannMSFT** [comment]
Version 1.3.0 is coming up on 2 years old. We have not seen any freezing problems in the latest version, 1.12.0. @rtelson if you are still having problems please open a new issue. Thanks.
